### DISCUSSION TITLE
"n.tag.map is not a function or its return value is not iterable"

### DISCUSSION THREAD

**Discussion by jacksbuilds** [ObsidianPublisher/obsidian-github-publisher#158] [Q&A — Plugin]
Hello,

I appreciate your work on this plugin, it's been very helpful in my testing! I think I may have encountered a bug.

I have a couple files that are not wanting to upload properly. My gut feeling is because the notes contain codeblocks, and those codeblocks may be confusing the GitHub API?

- Command used: 
	- [X] SHARE ON FILE
	- [X] SHARE ALL SHARED FILE
	- [X] Update the repository with new file only
	- [ ] Update the repository with edited file only
	- [ ] Update the repository with edited and new file
	- [ ] Clean the repository
- My plugin data : [gist here](https://gist.github.com/jdoescyber/d270b6891359f8c33b88cff619295440)
- The file I try to send: [another gist here](https://gist.github.com/jdoescyber/f316b300c5ecca793e3c8ff32e7adcca)
- [X] I already checked my repository and it's valid 
- [X] My repository exists (link to the repo: [here](https://github.com/jdoescyber/open-notes/tree/main) )
- [X] I already tried to send an example file to the repo 
- Screenshot to the log: [here, on Imgur](https://i.imgur.com/VZEmhNX.png)
 
Edit:

I should clarify that this error seems to result in the file not being uploaded to the repository.

**Comment by Mara-Li** [comment]
Hello!
It's a bug given from the Obsidian API, meaning your tags are wrongly formatted!

Obsidian API is pretty strict with tags, so you can use linter plugin to format them.

You need to use `tag:[tag,tag]`

I can also add some checker to prevent these bug!

**Comment by Mara-Li** [comment]
Please use #159 to follow this bug report :)
